docs/guides/_common/ory-cli.mdx

```diff
@@ -1,18 +1,21 @@
-The Ory CLI includes useful functionality to manage your Ory Cloud Project. But
-that is not why we require it in this guide!
+Ory CLI provides a convenient way to configure and manage Ory Cloud projects. Additionally, the CLI 
+contains Ory Proxy - a reverse proxy that rewrites cookies to match the domain your application is currently on. 
 
-Ory's philosophy is to make hard things easy for you. For this reason, Ory has
-deployed measures against all
-[OWASP Top 10](https://owasp.org/www-project-top-ten/) and implements the
```

Comment:
In the past, people have been quite receptive to claims that we take care of many security concerns (such as the ones in OWASP). For example, Anti-CSRF measures are annoying when setting it up, but life saving when protecting against CSRF attacks. It doesn't have to be in the first paragraph, but I think we should at least mention that "Ory keeps you safe", and not just "Ory sets the domain right" :)

Reply:
I can do that if you REALLY want to see it mentioned here, but consider this - this is a snippet used on a very specific set of documents only. These documents are examples that show you very simple apps consuming Ory. The question here is - why do I, the user, even need the CLI to work with these examples? 
The way I structured the description aims to answer this question and bring attention to two things that are the most important in that context: 
- the fact that the proxy "connects you to the cloud so you don't need to install anything else on your machine to see the setup working" 
- the fact that the endpoints are mirrored on the same domain and cookie origin is setup correctly thanks to that

That's why I cut the "Ory keeps you safe" elements :)

Reply:
That makes sense, maybe we can move it to a separate document then? For example `Security Architecture` or something like that? And have an overview of the different security relevant topics for nerds? Similar to: https://www.ory.sh/docs/hydra/security-architecture

Reply:
Yes, I'd rather do it this way. :)